app/views/users/index.html.erb

```diff
@@ -1 +1,10 @@
-<h1>A list of all users</h1>
\ No newline at end of file
+<h1>A list of all users</h1>
+<% @users.each do |user| %>
```

Comment:
Every user on `app/views/index.html.erb`should have a link and clicking on it should take the user to the user's show page.